app/screens/seed-words-export-screen/seed-words-export-screen.tsx

```diff
@@ -0,0 +1,263 @@
+import { ChainStore } from "../../models/chain-store"
+import { logError } from "../../utils/error"
+import { sizes } from "../../components/text/text.sizes"
+import { Icon } from "../../components/icon"
+
+const Screen = styled(ScreenBase)`
+  flex: 1;
+`
+
+const ScrollView = styled.ScrollView`
+  flex: 1;
+  padding: ${({ theme }) => theme.spacing.md};
+`
+
+const NoticeView = styled.View`
+  margin-bottom:  ${({ theme }) => theme.spacing.xl};
+  padding: ${({ theme }) => theme.spacing.lg};
+  background-color: ${({ theme }) => theme.color.palette.white};
+  border-radius: 14px;
+`
+
+const NoticeTitle = styled(Text)`
+  color: ${({ theme }) => theme.color.palette.angry};
+  text-align: center;
+  font-size: ${({ theme }) => theme.text.size.xl};
+  font-weight: 600;
+`
+
+const NoticeDescription = styled(Text)`
+  margin-top: ${({ theme }) => theme.spacing.sm};
+  font-size: ${({ theme }) => theme.text.size.md};
+  color: ${({ theme }) => theme.color.palette.angry};
+`
+
+const PasswordInputLabel = styled(Text)`
+  color: ${({ theme }) => theme.color.palette.white};
+`
+
+const PasswordInputField = styled.TextInput`
+  margin-top: ${({ theme }) => theme.spacing.sm};
+  padding: ${({ theme }) => theme.spacing.sm};
+  border-radius: 8px;
+  background-color: ${({ theme }) => theme.color.palette.white};
+  text-align: center;
+`
+
+const AuthenticateButton = styled(Button)`
+  margin-top:  ${({ theme }) => theme.spacing.md};
+`
+
+const SeedWordsContainer = styled.View`
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: ${({ theme }) => theme.spacing.md};
+  background-color: ${({ theme }) => theme.color.palette.white};
+  border-radius: 14px;
+`
+
+const SeedWordText = styled.Text`
+  padding: ${({ theme }) => theme.spacing.xs};
+  padding-left: ${({ theme }) => theme.spacing.sm};
+  padding-right: ${({ theme }) => theme.spacing.sm};
+  border-radius: 4px;
+  background-color: ${({ theme }) => theme.color.palette.greyf2};
+  margin: 4px;
+`
+
+const CopySeedWordsButton = styled(Button)`
+  margin-top:  ${({ theme }) => theme.spacing.md};
+`
+
+const ErrorView = styled.View`
+  flex-direction: row;
+  margin-top:  ${({ theme }) => theme.spacing.md};
+`
+
+export interface SeedWordsExportScreenProps extends NavigationStackScreenProps<{}> {
+  chainStore: ChainStore
+  userStore: UserStore
+}
+
+@inject("chainStore", "userStore")
+@observer
+export class SeedWordsExportScreen extends React.Component<SeedWordsExportScreenProps, {}> {
+  state = {
+    isLoading: false,
+    isConfirmed: false,
+    isPasswordNeeded: false,
+    isCopied: false,
+    password: '',
+    seeds: '',
+    error: '',
+  }
+
+  authClient: AuthCoreAuthClient
+
+  get seedWords() {
+    return this.state.seeds.split(' ')
+  }
+
+  private goBack = () => {
+    this.props.navigation.goBack()
+  }
+
+  private onPressConfirmButton = async () => {
+    try {
+      this.setState({ isLoading: true })
+      const { accessToken } = this.props.userStore.authCore
```

Comment:
Should we force a new login here like in like-co widget?

Reply:
Added, need to update [react-native-authcore](https://gitlab.com/likecoin/react-native-authcore/-/merge_requests/7) as well